torch/distributed/distributed_c10d.py

```diff
@@ -1072,6 +1073,16 @@ def get_backend(group: Optional[ProcessGroup] = None) -> Backend:
     pg_store = _world.pg_map[pg] if pg in _world.pg_map else None
     return Backend(not_none(pg_store)[0])
 
+def _get_process_group_uid(pg: ProcessGroup) -> int:
+    backend = None
+    try:
+        backend = pg._get_backend(torch.device("cuda"))
+    except RuntimeError:
+        pass
```

Comment:
Shouldn't the exception at least be logged?

Reply:
All the RuntimeError exceptions in c10d during `_get_backend `are handled by 'pass' without logging. Just follow the same logic